CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+# Releasing 0.5.1 (unreleased)
+
+  * Drop support for Ruby 1.8.7
+  * voxpupuli namespace release
```

Comment:
wasn't 0.5.0 the voxpupuli n/s release? Or does it not count because the readme was not 100% up to date?